text/java/0000-oracle-jdk.md

```diff
@@ -0,0 +1,38 @@
+# Paketo Buildpacks Oracle JDK Buildpack
+
+## Summary
+
+A [Oracle JDK Buildpack](https://github.com/eddumelendez/oracle-jdk) for supplying Oracle's distribution of OpenJDK has been created by [@eddumelendez][@eddumelendez]. It is built using common tooling (libpak and libjvm) used by the Paketo Java sub-team and should be adopted into the Paketo Buildpacks Github Org.
```

Comment:
I'm curious about the name. What would you think about just `oracle` or `oracle-java` for the name?

I don't know if they ever will, but my concern with the present name is that if Oracle starts releasing JREs, the name would be a little misleading.

Reply:
I would choose just `oracle`

Reply:
OK, let's go with `paketo-buildpacks/oracle` then.